client/src/components/requester_order/partial-details.html

```diff
@@ -1,15 +1,13 @@
 <div id="requester-order-details">
   <h1> Sit tight, you will have a queue hero soon! </h1>
-
-  <div>
-    <button ng-click="ctrl.cancelOrder()">Cancel Order</button>
+  <div class="order-details">
+    <h3 class="your-order"> Your order </h3>
+    <div class="order-line"> Meeting location: {{ ctrl.order.vendor }} </div>
+    <div class="order-line"> Item: {{ ctrl.order.item }} </div>
+    <div class="order-line"> Additional Requests: {{ ctrl.order.additionalRequests }} </div>
+    <div class="order-line"> Meeting time: {{ ctrl.order.meetingTime | date : 'EEE d MMM h:mm a' }} </div>

```

Comment:
This shows the date as `Wed 14 Oct 4:06 PM` I think it's clearer if it's `Wed. Oct 14, 4:06 PM`